.circleci/config.yml

```diff
@@ -45,14 +45,34 @@ jobs:
       - install-bazel-linux-rbe
       - checkout
       - run: bazel run @graknlabs_build_tools//checkstyle:test-coverage
-      - run: |
-          SONARCLOUD_CODE_ANALYSIS_CREDENTIAL=$SONARCLOUD_CREDENTIAL \
-            bazel run @graknlabs_build_tools//sonarcloud:code-analysis -- \
-            --organisation $SONARCLOUD_ORGANISATION --project-key $SONARCLOUD_PROJECT \
-            --commit-id=$CIRCLE_SHA1 --branch=$CIRCLE_BRANCH
       - run-bazel-rbe:
           command: bazel build //...
-  
+  code-analysis-pr:
+    machine: true
+    working_directory: ~/grakn
+    steps:
+      - install-bazel-linux-rbe
+      - checkout
+      - run: |
+          export SONARCLOUD_CODE_ANALYSIS_CREDENTIAL=$SONARCLOUD_CREDENTIAL
```

Comment:
Public contributors also have to have their own `$SONARCLOUD_CREDENTIAL`?

Reply:
@haikalpribadi yes